src/test/java/graphql/annotations/GraphQLObjectTest.java

```diff
@@ -622,7 +622,7 @@ public void queryOptional() {
         GraphQLObjectType object = GraphQLAnnotations.object(OptionalTest.class);
         GraphQLSchema schema = newSchema().query(object).build();
 
-        GraphQL graphQL = GraphQL.newGraphQL(schema).queryExecutionStrategy(new EnhancedExecutionStrategy()).build();
```

Comment:
Why did you remove the queryExecutionStrategy?

Reply:
He may removed it during development but forgot to put it back with its third commit. 

Also https://github.com/graphql-java/graphql-java-annotations/pull/105/files#diff-9599d6f1ee8ac402b47f46e351eead38L644 and https://github.com/graphql-java/graphql-java-annotations/pull/105/files#diff-9599d6f1ee8ac402b47f46e351eead38L675

Reply:
Yes, I restored an EnhancedExecutionStrategy for gql4.2, but did not put it back on tests where it was not needed - it's only used for relay mutations, so kept it only in RelayTest. I could restore it in these tests also, but it won't change much.

Reply:
If its not necessary, no need